beluga/include/beluga/sensor/data/dense_grid2_mixin.hpp

```diff
@@ -71,16 +57,16 @@ namespace beluga {
 
 /// Dense 2D grid base type.
 /**
- * When instantiated, it satisfies \ref DenseGrid2Page.
- *
- * \tparam Derived Concrete dense grid type. It must define
- * `Derived::width()`, `Derived::height()`, `Derived::resolution()`,
- * `Derived::data_at(index)`, and `Derived::index_at(int, int)`
- * as described in \ref DenseGrid2Page.
+ * \tparam Mixin Mixin type
  */
-template <typename Derived>
-class BaseDenseGrid2 : public BaseRegularGrid2<Derived> {
+template <typename Mixin>
+class DenseGrid2Mixin : public Mixin {
```

Comment:
@glpuga nit^2: do we need to (over)specify it's a mixin by tucking `mixin` at the end of the class name? Same elsewhere.